src/main/java/graphql/testrunner/service/CommandExecutorService.java

```diff
@@ -73,7 +73,7 @@ private void writeOutput(Process process, List<String> command) throws IOExcepti
                 String errorLog = readOutput(process, Process::getErrorStream);
                 LOGGER.log(Level.SEVERE, "Error exit code on command : {0}", command);
                 LOGGER.log(Level.SEVERE, "Error message: {0}", errorLog);
-                throw new TestRunnerException();
+                throw new TestRunnerException("Command failed to execute.");
```

Comment:
please pass the error message in here too

Reply:
done